packages/@o3r/eslint-config/src/rules/typescript/unicorn.cjs

```diff
@@ -26,6 +26,7 @@ const config = [
       'unicorn/no-array-reduce': 'off',
       'unicorn/no-null': 'off',
       'unicorn/no-typeof-undefined': 'off',
+      'unicorn/prefer-code-point': 'off',
```

Comment:
why is it disabled?

Reply:
I didn't like the fact that there was no auto-fix, it's an easy search and replace though

Reply:
After discussion with the team we keep this rule set to `off`